src/auxiliary_functions.py

```diff
@@ -0,0 +1,63 @@
+
+from models import ClassificationResponse, Metadata, OtherProvidersClassifications
+from errors import ApkAnalyzerError
+from apk_analyzer import ApkAnalyzer
+from classifier import Classifier
+import time
+import logging
+import zipfile
+from io import BytesIO
+import os
+from dotenv import load_dotenv
+
+if os.getenv("DOCKER_ENV"):
+    load_dotenv(".docker.env")
+else:
+    load_dotenv(".env")
+
+CSV_PATH = os.getenv('CSV_PATH_FROM_SRC')
+
+class Auxiliary:
+    def __init__(self, apk_analyzer, redis_cache):
+        self.apk_analyzer = apk_analyzer
+        self.redis_cache = redis_cache
+
+    @staticmethod
```

Comment:
in utils file, you dont need to have these function under a class. they can be global functions:
`tracer()`, `is_apk()`